tests/integration/release/conftest.py

```diff
@@ -56,6 +59,12 @@
 
 
 def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--output-dir",
+        action="store",
+        default=CLI_DEFAULT_OUTPUT_DIR,
```

Comment:
Does this default mean that every test we run (even if not a release test) will get saved out to this dir? Or no b/c it's in the `integration/release/conftest`?